### DISCUSSION TITLE
Google Colab link isn't accessible

### DISCUSSION THREAD

**Discussion by mayaniyogi** [mir-group/nequip#436] [Q&A]
In the tutorial section, there's a link to a Google Colab notebook, but when I try to click the link, it says Access Denied. However I see that 3 weeks ago it was possible to open it, so I'm wondering if I need to change something on my end or if it's geniunely an access issue. Any advice would be appreciated!
<img width="763" alt="Screenshot 2024-06-19 at 16 10 14" src="https://github.com/mir-group/nequip/assets/76263542/c7e15741-0446-4c55-bdd8-8b348e300519">

**Comment by cw-tan** [comment]
Hi @mayaniyogi,

Thank you for your interest in NequIP and apologies for the inconvenience. As mentioned in https://github.com/mir-group/nequip/issues/437, the Google account hosting the Colab tutorial expired hence the access issue. 

For learning how to use `nequip`, I'd recommend reading through `nequip/configs/example.yaml` [here](https://github.com/mir-group/nequip/blob/main/configs/example.yaml) and `nequip/configs/full.yaml` [here](https://github.com/mir-group/nequip/blob/main/configs/full.yaml), and trying to train models with those config files.

**Reply by catastropiyush** [reply]
Hello,
       I just started out with using NequIP. 
      `$ nequip-train configs/example.yaml` After running this I get the following files in the directory.
      
![image](https://github.com/mir-group/nequip/assets/61222720/b59b19e6-e965-4f48-ad3b-bb2320b7223a)
But how do I deploy the trained model for LAMMPS?
`nequip-deploy build path/to/training/session/ path/to/deployed.pth`
What should one put in `path/to/training/session` the directory that contains the `.yaml` file?
```
pair_style	nequip
pair_coeff	* * deployed.pth <NequIP type for LAMMPS type 1> <NequIP type for LAMMPS type 2> ...
```
What are LAMMPS type 1 and LAMMPS type 2 here?

I apologize if the questions are too basic.
Thank you for your time

**Comment by Linux-cpp-lisp** [comment]
For an updated tutorial, please see https://github.com/mir-group/nequip/?tab=readme-ov-file#tutorial.
